src/llaminator.ts

```diff
@@ -16,6 +16,7 @@
 
 import { render } from 'lit-html';
 
+import { easterEggs } from './easter-basket';
```

Comment:
optional nit: I feel that this should be kEasterEggs or something as it's a constant. Our style guide seems quite.. undecided about this, is that intentional?

Reply:
sure, I don't have strong feelings here